src/NServiceBus.Core/Sagas/SagaPersistenceBehavior.cs

```diff
@@ -296,6 +296,11 @@ IContainSagaData CreateNewSagaEntity(SagaMetadata metadata, IInvokeHandlerContex
 
             if (lookupValues.TryGet(sagaEntityType, out var value))
             {
+                if (value.PropertyValue == null)
+                {
+                    throw new Exception($"Message of type {context.MessageMetadata.MessageType.FullName} mapped to the saga of type {metadata.SagaType.FullName} has attempted to assigned a null to the correlation property {value.PropertyName}.");
```

Comment:
```suggestion
                    throw new Exception($"Message {context.MessageMetadata.MessageType.FullName} mapped to the saga of type {metadata.SagaType.FullName} has attempted to assign null value to the correlation property {value.PropertyName}. Correlation properties cannot be assigned null values.");
```

Reply:
👍 I'll change that via manual change as it has to change in two places